apps/backend/api/graphql/schema.graphql

```diff
@@ -2093,6 +2139,8 @@ type Post {
   announcement: Boolean
   # Number of attachments on this post
   attachmentsTotal: Int
+  # Budget for project or submission posts
+  budget: String
```

Comment:
is this a string for a url?

Reply:
no just for text content. they can describe their budget however they want